AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/helper/AndroidManifestFinder.java

```diff
@@ -229,12 +229,15 @@ private void findPossibleLocationsV32(String basePath, String variantPart, List<
 				variantPart = sb.toString();
 			}
 
-			String processManifest = "process" + upperCaseFirst(variantPart) + "Manifest";
-			String possibleLocation = BUILD_TOOLS_V32_MANIFEST_PATH + "/" + variantPart + "/" + processManifest + "/merged";
-			File variantDir = new File(basePath + possibleLocation);
-			if (variantDir.isDirectory()) {
-				possibleLocations.add(possibleLocation);
-				addPossibleSplitLocations(basePath, possibleLocation, possibleLocations);
+			String possibleLocationWithoutProcessManifest = BUILD_TOOLS_V32_MANIFEST_PATH + "/" + variantPart;
+			String possibleLocationWithProcessManifest = possibleLocationWithoutProcessManifest + "/process" + upperCaseFirst(variantPart) + "Manifest/merged";
+			if (new File(basePath, possibleLocationWithoutProcessManifest).isDirectory()) {
```

Comment:
Should we extract this to a method? The two ifs are doing the same if I am seeing it right.